src/Tools/dotnet-monitor/InProcessFeatures/InProcessFeatureOptionsBinder.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.Diagnostics.Monitoring.Options;
+using Microsoft.Extensions.Configuration;
+
+namespace Microsoft.Diagnostics.Tools.Monitor
+{
+    internal static class InProcessFeatureOptionsBinder
+    {
+        public static void BindEnabled(IInProcessFeatureOptions options, string configurationKey, IConfiguration configuration, bool enabledByDefault)
+        {
+            // Get the value of InProcessFeatures:<Feature>:Enabled. This may have already
+            // been bound by the default options binding, but this makes it consistent for
+            // anything that passes in an unbound options instance and fulfills the complete
+            // expected behavior of binding the Enabled property.
+            options.Enabled = configuration
```

Comment:
Just to be clear, if the user says "InProcessFeatures": {}, the result will be enabled = true?

Reply:
No, that doesn't translate to anything in configuration. The minimal to enable in-process features is:
```
{
  "InProcessFeatures": {
    "Enabled": true
  }
}
```
This is the same as it was before.

However, I could add the ability to allow the following to enable all in-process features:
```
{
  "InProcessFeatures": true
}
```

Reply:
Chatted offline... the "Enable" properties are good enough for now. I've removed the `InProcessFeatures:<Feature> = true/false` ability. We can add that back later if there is demand for this simplified experience.